schemas/tlc/1.1.1/sxl.yaml

```diff
@@ -218,11 +218,12 @@ objects:
             description: |-
               False: Controller is not in start up mode
               True: Controller is currently in start up mode
-      S0006:
+            S0006:
```

Comment:
This looks like an unintended indentation and breaks the format

Reply:
fixed